blockchain/blockdao/blockindexer.go

```diff
@@ -67,9 +67,14 @@ func (bic *BlockIndexerChecker) CheckIndexer(ctx context.Context, indexer BlockI
 	if tipHeight > daoTip {
 		return errors.New("indexer tip height cannot by higher than dao tip height")
 	}
-	tipBlk, err := bic.dao.GetBlockByHeight(tipHeight)
-	if err != nil {
-		return err
+	var tipBlk *block.Block
+	if tipHeight > 0 {
```

Comment:
this is not necessary? if `tipHeight == 0`, it will return genesis block inside `bic.dao.GetBlockByHeight()`?